.github/workflows/release.yml

```diff
@@ -0,0 +1,41 @@
+name: Auto-release on PR merge
```

Comment:
could it be also used/allowed to be triggered manually among actions on current shape of master branch?

Reply:
@yarikoptic What do you mean by "on current shape of master branch"?

Reply:
Just that we could trigger release of current master version without PR

Reply:
f4b1a56 should do it.